tests/integration/fuzz/lockup-base/withdraw.t.sol

```diff
@@ -222,7 +222,7 @@ abstract contract Withdraw_Integration_Fuzz_Test is Integration_Test {
         address expectedNFTOwner = users.recipient;
         assertEq(actualNFTowner, expectedNFTOwner, "NFT owner");
 
-        // It should update the aggrate balance.
+        // It should update the aggregate balance.
```

Comment:
Since its fuzz test, shouldn't we use `Asset that` approach here similar to your refactor in cancel test?

I like the "asset that" approach in fuzz tests and "it should" approach in concrete tests.

Reply:
It's up to you.

Reply:
I will leave it unchanged then.